examples/AdvancedOperations/AddAppCampaign.php

```diff
@@ -0,0 +1,443 @@
+        int $customerId,
+        string $campaignResourceName
+    ) {
+        $campaignCriterionOperations = [];
+
+        // Creates the location campaign criteria.
+        // Besides using location_id, you can also search by location names from
+        // GeoTargetConstantService.suggestGeoTargetConstants() and directly
+        // apply GeoTargetConstant.resource_name here. An example can be found
+        // in GetGeoTargetConstantByNames.php.
+        $locationIds = [
+            21137, // California
+            2484   // Mexico
+        ];
+        foreach ($locationIds as $locationId) {
+            // Creates a campaign criterion.
+            $campaignCriterion = new CampaignCriterion([
+                'campaign' => new StringValue(['value' => $campaignResourceName]),
+                'type' => CriterionType::LOCATION,
+                'location' => new LocationInfo([
+                    'geo_target_constant' => new StringValue([
+                        'value' => ResourceNames::forGeoTargetConstant($locationId)
+                    ])
+                ])
+            ]);
+
+            // Creates a campaign criterion operation.
+            $campaignCriterionOperation = new CampaignCriterionOperation();
+            $campaignCriterionOperation->setCreate($campaignCriterion);
+
+            $campaignCriterionOperations[] = $campaignCriterionOperation;
+        }
+
+        // Creates the language campaign criteria.
+        $languageIds = [
+            1000, // English
+            1003  // Spanish
+        ];
+        foreach ($languageIds as $languageId) {
+            // Creates a campaign criterion.
+            $campaignCriterion = new CampaignCriterion([
+                'campaign' => new StringValue(['value' => $campaignResourceName]),
+                'type' => CriterionType::LANGUAGE,
+                'language' => new LanguageInfo([
+                    'language_constant' => new StringValue(['value' =>
+                        ResourceNames::forLanguageConstant($languageId)
+                    ])
+                ])
+            ]);
+
+            // Creates a campaign criterion operation.
+            $campaignCriterionOperation = new CampaignCriterionOperation();
+            $campaignCriterionOperation->setCreate($campaignCriterion);
+
+            $campaignCriterionOperations[] = $campaignCriterionOperation;
+        }
+
+        // Submits the criteria operations.
+        $campaignCriterionServiceClient = $googleAdsClient->getCampaignCriterionServiceClient();
+        $response = $campaignCriterionServiceClient->mutateCampaignCriteria(
+            $customerId,
+            $campaignCriterionOperations
+        );
+
+        $createdCampaignCriteriaResourceName = $response->getResults()[0]->getResourceName();
+        printf("Created Campaign Criteria '%s'%s", $createdCampaignCriteriaResourceName, PHP_EOL);
+    }
+
+    /**
+     * Creates an ad group and associated app ap for a given campaign.
```

Comment:
App Ad.

Also, scan to make sure you are using the same casing everywhere. E.g. I see both **c**ampaign and **C**ampaign in the same code example. Make sure you use a consistent style.

Reply:
ad should be small letter. Only "App" should be in a capital letter.

Reply:
ad should be small letter. Only "App" should be in a capital letter.